pinot-core/src/main/java/com/linkedin/pinot/core/startree/OffHeapStarTreeBuilder.java

```diff
@@ -493,28 +497,28 @@ private void printTree(StarTreeIndexNode node, int level) {
     BiMap<Integer, String> inverse = dimensionNameToIndexMap.inverse();
     String dimName = "ALL";
     Object dimValue = "ALL";
-    if (node.getDimensionName() != StarTreeIndexNode.all()) {
+    if (node.getDimensionName() != StarTreeIndexNodeInterf.ALL) {
       dimName = inverse.get(node.getDimensionName());
     }
-    if (node.getDimensionValue() != StarTreeIndexNode.all()) {
+    if (node.getDimensionValue() != StarTreeIndexNodeInterf.ALL) {
       dimValue = dictionaryMap.get(dimName).inverse().get(node.getDimensionValue());
     }
 
     String formattedOutput = Objects.toStringHelper(node).add("nodeId", node.getNodeId())
         .add("level", level).add("dimensionName", dimName).add("dimensionValue", dimValue)
         .add("childDimensionName", inverse.get(node.getChildDimensionName()))
-        .add("childCount", node.getChildren() == null ? 0 : node.getChildren().size())
+        .add("childCount", node.getNumChildren())
```

Comment:
Does num refer to the number of children (count) or number of children (child id)? Since it looks like it's the former, I'd call this `getChildrenCount()`.